libp2p/NodeTable.cpp

```diff
@@ -415,19 +416,21 @@ void NodeTable::onReceived(UDPSocketFace*, bi::udp::endpoint const& _from, bytes
 				bool found = false;
 				EvictionTimeout evictionEntry;
 				DEV_GUARDED(x_evictions)
-					for (auto it = m_evictions.begin(); it != m_evictions.end(); ++it)
-						if (it->first.first == in.sourceid && it->first.second > std::chrono::steady_clock::now())
+					if (m_evictions.count(in.sourceid))
```

Comment:
To make it more efficient use `unordered_map::find()`, then use the returned iterator in place of all `m_evictions[in.sourceid]` & `m_evictions.erase(in.sourceid)`

Reply:
i'll fix it